// nativescript-ar/ar.android.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var ar_common_1 = require("./ar-common");
var AR = (function (_super) {
    __extends(AR, _super);
    function AR() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
    AR.prototype.togglePlaneVisibility = function (on) {
    };
    AR.prototype.togglePlaneDetection = function (on) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.toggleStatistics = function (on) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.setDebugLevel = function (to) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.reset = function () {
        throw new Error("Method not implemented.");
    };
    AR.prototype.addModel = function (options) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.addBox = function (options) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.addSphere = function (options) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.addText = function (options) {
        throw new Error("Method not implemented.");
    };
    AR.prototype.addTube = function (options) {
        throw new Error("Method not implemented.");
    };
    return AR;
}(ar_common_1.AR));
exports.AR = AR;
